src/html_json.js

```diff
@@ -112,26 +207,28 @@ module.exports.main = async (context, action) => {
     owner, repo, ref, path,
   } = action.request.params;
 
-  // TODO: this should be loaded once or even passed in context
-  const config = await loadConfig(owner, repo, ref);
+  // Use a file loader if this is a local setup
+  const loader = (owner === 'helix') ? fileLoader : repoLoader;
```

Comment:
- how does the `owner===helix` indicate a local setup?
- why using a local setup at all? if it's just for development, I would use a `develop` flag.

Reply:
local setup meaning it was started by `helix-simulator`, in `HelixProject.emulateGit`